src/scene_graph/camera_node.cpp

```diff
@@ -17,11 +18,101 @@ namespace wr
 		DirectX::XMVECTOR forward = DirectX::XMVector3Normalize(m_transform.r[2]);
 		DirectX::XMVECTOR right = DirectX::XMVector3Normalize(m_transform.r[0]);
 
-		m_view = DirectX::XMMatrixLookAtRH(pos, DirectX::XMVectorAdd(forward, pos), up);
-		m_projection = DirectX::XMMatrixPerspectiveFovRH(m_fov, m_aspect_ratio, 1, 125);
+		m_view = DirectX::XMMatrixLookAtRH(pos, DirectX::XMVectorAdd(pos, forward), up);
+		m_projection = DirectX::XMMatrixPerspectiveFovRH(m_fov, m_aspect_ratio, m_frustum_near, m_frustum_far);
```

Comment:
Can you please give default values of 1 and 125 to near and far in the constructor? So that we make sure ray tracing doesn't break in master because of wrong camera settings